modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/OperationMessageProcessor.java

```diff
@@ -91,7 +92,8 @@
  *
  * @since 3.7.0
  */
-public class OperationMessageProcessor extends ExtensionComponent implements Processor, EntityMetadataProvider, Lifecycle {
+public class OperationMessageProcessor extends ExtensionComponent implements Processor, EntityMetadataProvider, Lifecycle,
+    Interceptable {
```

Comment:
You shouldn't treat this one the same way to treat the others. Interceptable should probably be more than just a marker interface and give room for different components to be intercepted in their own way.